PhysicsTools/NanoAOD/python/jetsAK4_CHS_cff.py

```diff
@@ -171,11 +171,15 @@
         hfcentralEtaStripSize = Var("userInt('hfJetShowerShape:centralEtaStripSize')", int, doc="eta size of the central tower strip in HF (noise discriminating variable) "),
         hfadjacentEtaStripsSize = Var("userInt('hfJetShowerShape:adjacentEtaStripsSize')", int, doc="eta size of the strips next to the central tower strip in HF (noise discriminating variable) "),
         nConstituents = Var("numberOfDaughters()","uint8",doc="Number of particles in the jet"),
+        chMultiplicity = Var("chargedMultiplicity()","uint8",doc="Number of charged particles in the jet"),
```

Comment:
are these changes intended? I didn't expect to see changes to the nano content in this PR